bundles/org.openhab.binding.ecovacs/README.md

```diff
@@ -0,0 +1,175 @@
+# Ecovacs Binding
+
+This binding provides integration for vacuum cleaning / mopping robots made by Ecovacs (<https://www.ecovacs.com/>).
+It discovers devices and communicates to them by using Ecovacs' cloud services.
+
+## Supported Things
+
+- Ecovacs cloud API (`ecovacsapi`)
+- Vacuum cleaner (`vacuum`)
+
+At this point, the following devices are fully supported and verified to be working:
+
+- Deebot OZMO 900/905
+- Deebot OZMO 920
+- Deebot OZMO 930
+- Deebot OZMO 950
+- Deebot OZMO Slim 10/11
+- Deebot N8 series
+
+The following devices will likely work because they are using similar protocols as the above ones:
+
+- Deebot 600/601/605
+- Deebot 900/901
+- Deebot OZMO 610
+- Deebot 710/711/711s
+- Deebot OZMO T5
+- Deebot (OZMO) T8 series
+- Deebot T9 series
+- Deebot Slim 2
+- Deebot N3 MAX
+- Deebot N7
+- Deebot U2 series
+- Deebot X1 Omni
+
+## Discovery
+
+At first, you need to manually create the bridge thing for the cloud API.
+Once that is done, the supported devices will be automatically discovered and added to the inbox.
+
+## Thing Configuration
+
+For the cloud API thing, the following parameters must be configured:
+
+| Config    | Description                                                                                                                   |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------|
+| email     | The email address you used when registering the Ecovacs cloud account                                                         |
+| password  | The cloud account password                                                                                                    |
+| continent | The continent you are residing on, or 'World' if none matches. This is used to select the correct cloud server to connect to. |
+
+For the vacuum things, there is no required configuration (when using discovery). The following parameters exist:
+
+| Config       | Description                                                                                                                   |
+|--------------|-------------------------------------------------------------------------------------------------------------------------------|
+| serialNumber | Required: The device's serial number as printed on the barcode below the dust bin. Filled automatically when using discovery. |
+| refresh | Refresh interval for polled data (see below), in minutes. By default set to 5 minutes.                                             |
```

Comment:
```suggestion
| refresh      | Refresh interval for polled data (see below), in minutes. By default set to 5 minutes.                                        |
```

Reply:
Thanks, stupid me :facepalm:

Reply:
Can we align it fully? 🙂

Reply:
Yes ... looking at https://github.com/openhab/openhab-addons/pull/12231/commits/ba11c9766bf44b72ca7caea71d49da6571484115 it's precisely what I did ;-)

Reply:
> Yes ... looking at https://github.com/openhab/openhab-addons/pull/12231/commits/ba11c9766bf44b72ca7caea71d49da6571484115 it's precisely what I did ;-)

Ah, missed that, sorry.